pkg/tool/kubectl.go

```diff
@@ -54,66 +43,91 @@ func (k Kubectl) DeleteNamespace(namespace string) {
 		fmt.Println("Error deleting pods:", err)
 	}
 
-	time.Sleep(3 * time.Second)
+	fmt.Println("Force-deleting pvcs...")
```

Comment:
if we delete the pvc/pvs/pods after we tried to delete the namespace, did we not fall in the orphan case? that we cannot remove using kubectl?

Reply:
Deleting the Helm release should already do this. Then, deleting the namespace should also delete all resources within it. If some resources are still stuck terminating after that, we force-delete them. Then we wait again a little while to see if the namespace terminates before we remove its finalizers and force-delete it.

Reply:
I had this same question. Sounds like a sane strategy 👍